src/index.js

```diff
@@ -17,22 +15,21 @@ class LevelDatastore {
       database = opts.db
       delete opts.db
     } else {
-      // Default to leveldown db
-      database = require('leveldown')
+      database = require('level')
     }
 
-    this.db = levelup(
-      encode(database(path), { valueEncoding: 'binary' }),
-      Object.assign({}, opts, {
-        compression: false // same default as go
-      }),
-      (err) => {
-        // Prevent an uncaught exception error on duplicate locks
-        if (err) {
-          throw err
-        }
-      }
-    )
+    this.db = database(path, {
+      ...opts,
+      valueEncoding: 'binary',
+      compression: false // same default as go
+    })
+
+    this.db.put = promisify(this.db.put)
```

Comment:
Why is this needed? In case a custom db (opts.db) is passed in?

Reply:
I was seeing errors like `'get() requires a callback argument'` ...and now I'm not.  Have removed.